### PR TITLE
NISZ.AlairoAlkalmazas_KEAASZ version 1.33.0

### PR DESCRIPTION
**Pull request by diakritikus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/35388)

### CODE REVIEW

**Comment by diakritikus** [comment]
Can you please add the /S option when executing this .exe installer?
It will make the install go silently without user interaction.

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
+PackageVersion: 1.33.0
+PackageLocale: hu-HU
+Publisher: NISZ
+PackageName: Kormanyzati Elektronikus Alairo es Alairas-ellenorzo Szoftver
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml:8]:
```suggestion
PackageName: KEAASZ
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
+PackageVersion: 1.33.0
+PackageLocale: hu-HU
+Publisher: NISZ
+PackageName: Kormanyzati Elektronikus Alairo es Alairas-ellenorzo Szoftver
+License: Propriatery
+ShortDescription: KEAASZ
+Moniker: KEAASZ
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml:11]:
```suggestion
Moniker: keaasz
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
+PackageVersion: 1.33.0
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.installer.yaml:7]:
```suggestion
- Architecture: x64
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.installer.yaml:4]:
```suggestion
PackageIdentifier: NISZ.KEAASZ
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml:4]:
```suggestion
PackageIdentifier: NISZ.KEAASZ
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.yaml:4]:
```suggestion
PackageIdentifier: NISZ.KEAASZ
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
+PackageVersion: 1.33.0
+PackageLocale: hu-HU
+Publisher: NISZ
+PackageName: Kormanyzati Elektronikus Alairo es Alairas-ellenorzo Szoftver
+License: Propriatery
+ShortDescription: KEAASZ
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml:10]:
```suggestion
ShortDescription: A KEAASZ egy kormányzati elektronikus aláírás és aláírás-ellenőrző szoftver.
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
+PackageVersion: 1.33.0
+PackageLocale: hu-HU
+Publisher: NISZ
+PackageName: Kormanyzati Elektronikus Alairo es Alairas-ellenorzo Szoftver
+License: Propriatery
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.locale.hu-HU.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.installer.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NISZ.AlairoAlkalmazas_KEAASZ
+PackageVersion: 1.33.0
```
Comment (ItzLevvie) [inline manifests/n/NISZ/AlairoAlkalmazas_KEAASZ/1.33.0/NISZ.AlairoAlkalmazas_KEAASZ.installer.yaml:5]:
```suggestion
PackageVersion: 1.33.0
InstallerSuccessCodes:
- 1223
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
> Can you please add the /S option when executing this .exe installer? It will make the install go silently without user interaction.

It's automatically set when `InstallerType` is set to `nullsoft`.

**Pull request closed without merging**